src/SFA.DAS.EmployerFinance.UnitTests/Models/AccountTests.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using AutoFixture;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.EmployerFinance.Models;
+using SFA.DAS.Testing;
+
+namespace SFA.DAS.EmployerFinance.UnitTests.Models
+{
+    [TestFixture]
+    [Parallelizable]
+    public class AccountTests : FluentTest<AccountTestsFixture>
+    {
+        #region UpdateName
```

Comment:
personally I am not a fan of regions, but I can see this class was a pain, and leaving it un-regioned would result in a very large amount to code to scroll through. Is it worth at separating the fixture and test classes into different files? I know we have not done this elsewhere but given the size of this class it might be worth it.

Reply:
We could potentially also split the `AccountTests` class into 3 separate classes, one for each method under test. That would negate the need for the regions.

I suggest we keep this test consistent with the other tests in the solution for now, and we can decide as a team if we want to split _all_ the tests up like this, so that we maintain consistency.

Reply:
agreed